### DISCUSSION TITLE
1.1.0的insteall文件在电脑上打不开

### DISCUSSION THREAD

**Discussion by laoaideqi** [PenUniverse/PenMods#194] [你问我答] [answered]
文件刚打开就退出了python是3.12版本的下载文件运行不了用python打开显示语法问题

**Comment by Redbeanw44602** [comment]
Installer的安装包服务器炸了，可以先加tg群手动安装1.3.0预览版

**Reply by laoaideqi** [reply]
怎么加群啊

**Reply by laoaideqi** [reply]
还有没有其他方式下载啊

**Reply by Redbeanw44602** [reply]
群链接在README上写了....

**Reply by laoaideqi** [reply]
用不了tg注册不了一直在加载，还有没有其他的方式下载    修改版的安装在g3上太难受了

**Reply by Redbeanw44602** [reply, marked as answer]
> 用不了tg注册不了一直在加载，还有没有其他的方式下载    修改版的安装在g3上太难受了

1. 没有其他方式，用不了请自己解决
2. g3不支持1.3.0

**Reply by laoaideqi** [reply]
那g3用户怎么办

**Reply by XiexieLihua** [reply]
tg群号在哪？

**Reply by AronnaxM** [reply]
https://t.me/PenUniverse

**Reply by zhangMonday** [reply]
可否在这里简单介绍一下1.3.0更新内容，以及安装预览版后续能否通过ota升级为正式版？
由于在境内使用tg的某些限制，在社区内公布更多有关信息能方便考虑是否升级为预览版🙏

**Reply by Redbeanw44602** [reply]
> 可否在这里简单介绍一下1.3.0更新内容，以及安装预览版后续能否通过ota升级为正式版？
> 由于在境内使用tg的某些限制，在社区内公布更多有关信息能方便考虑是否升级为预览版🙏

你可以尝试突破某些限制，或者等待正式版释出...
